uploader.json

```diff
@@ -1351,7 +1351,7 @@
             "symbolic_name": "ceph_conf"
         },
         {
-            "file": "/var/log/ceph/ceph.log",
+            "file": "/var/log/ceph/ceph.log.\d$",
```

Comment:
Done

Reply:
@psachin you need to leave the `()*` in the filename or it will never check for the regex.  It should be `"file": "/var/log/ceph/().*ceph\\.log\\.\\d$"`.  I determined this by testing with the `_expand_path` referenced above.